src/node_version.h

```diff
@@ -40,6 +40,6 @@
  * an API is broken in the C++ side, including in v8 or
  * other dependencies.
  */
-#define NODE_MODULE_VERSION 45  /* io.js v3.0.0 */
+#define NODE_MODULE_VERSION 45  /* Node.js v3.0.0 */
```

Comment:
I think you should remove this change, it'll be updated properly prior to v4 anyway but it's currently accurate as is